browser/browser_js.go

```diff
@@ -0,0 +1,13 @@
+//go:build js
+// +build js
+
+package browser
+
+import (
+	"syscall/js"
+)
+
+func OpenUrl(url string) error {
+	js.Global().Call("open", url, "_blank", "")
+	return nil
+}
```

Comment:
In Safari, the open function must be invoked inside onClick event. However, Gio gets and releases this event. 

If `open` is called without direct user interaction, outside of onClick, it may not always work. It's inconsistent, so sometimes it works, but often it doesn't.

The "hack" requires the user to click anywhere in the page to open, it's not perfect, but works as fallback.